examples/virtual_keyboard.rs

```diff
@@ -10,7 +10,7 @@ fn main() -> std::io::Result<()> {
     keys.insert(Key::BTN_DPAD_UP);
 
     let mut device = VirtualDeviceBuilder::new()?
-        .name("Fake Keyboard")
+        .name(&"Fake Keyboard")
```

Comment:
Wait this `&` shouldn't be necessary, I think we should make `S: AsRef<str> + ?Sized`